repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,2836 @@
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: clinical_neurophysiology
+          locale.en.name: Clinical neurophysiology
+        /clinical_oncology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: clinical_oncology
+          locale.en.name: Clinical oncology
+        /clinical_pharmacology_and_therapeutics:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: clinical_pharmacology_and_therapeutics
+          locale.en.name: Clinical pharmacology and therapeutics
+        /clinical_radiology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: clinical_radiology
+          locale.en.name: Clinical radiology
+        /community_sexual_and_reproductive_health:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: community_sexual_and_reproductive_health
+          locale.en.name: Community sexual and reproductive health
+        /dermatology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: dermatology
+          locale.en.name: Dermatology
+        /emergency_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: emergency_medicine
+          locale.en.name: Emergency medicine
+        /endocrinology_and_diabetes:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: endocrinology_and_diabetes
+          locale.en.name: Endocrinology and diabetes
+        /forensic_psychiatry:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: forensic_psychiatry
+          locale.en.name: Forensic psychiatry
+        /gastroenterology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: gastroenterology
+          locale.en.name: Gastroenterology
+        /general_internal_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: general_internal_medicine
+          locale.en.name: General internal medicine
+        /general_practice__gp_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: general_practice__gp_
+          locale.en.name: General practice (GP)
+        /general_psychiatry:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: general_psychiatry
+          locale.en.name: General psychiatry
+        /general_surgery:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: general_surgery
+          locale.en.name: General surgery
+        /genitourinary_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: genitourinary_medicine
+          locale.en.name: Genitourinary medicine
+        /geriatric_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: geriatric_medicine
+          locale.en.name: Geriatric medicine
+        /haematology__doctor_:
```

Comment:
Can we remove the trailing `_` from the name and key ?

Reply:
Going through and changing all of these now